spec/factories/passengers.rb

```diff
@@ -7,40 +7,55 @@
     email { FFaker::Internet.email }
     phone { FFaker::PhoneNumber.short_phone_number }
     sequence(:spire) { |n| n.to_s.rjust(8, '0') + '@umass.edu' }
+    # active_status { 'active' }
```

Comment:
Commented-out?

Reply:
Sometimes I comment out lines to see whether they make a difference or not. This time, I forgot to remove it when it didn't.